index.html

```diff
@@ -0,0 +1,195 @@
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">
+                                Listen to the speakers from various countries about the messages.
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center hover-border">
+                            <img src="img/program_icon_02.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">GDG content</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">
+                                Google technologies in areas like Android, Flutter and so much more!
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center hover-border">
+                            <img src="img/program_icon_03.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">GDG Forum</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">
+                                Share your thoughts and opinions with experts for each topic.
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center hover-border">
+                            <img src="img/program_icon_04.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">Workshops</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"> 
+                                Full-day workshops, hands-on experiences, and debugging events.
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center hover-border">
+                            <img src="img/program_icon_05.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">Networking</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">Get opportunities to network with Coders around the world.</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="box">
+                    <a class="btn-lg text-light" href="#" role="button">Join devfest 2023</a>
+                </div>
```

Comment:
- [ ] While you did well making the previous requested change, a new issue was introduced as well. On the desktop view, the "Join devfest 2023" is missing. KIndly resolve this.

**This is how the section looks like:**

![image](https://user-images.githubusercontent.com/86318284/230317055-fb1210fa-27ed-4a49-9c80-6792ca421c4d.png)

Reply:
@Nemwel-Boniface  You are right. Working on it.